hasura/migrations/default/1704889531735_alter_table_public_user_drop_column_team_id/up.sql

```diff
@@ -0,0 +1 @@
+alter table "public"."user" drop column "team_id" cascade;
```

Comment:
this will break production, best to include a migration in this PR that creates the membership records for all existing users and in a separate PR we drop this attribute

Reply:
You are right, thank you!